src/planetsmath/functions.py

```diff
@@ -28,3 +28,17 @@ def sum_product(list):
     for item in list:
         product *= item
     return product
+
+
+def multiply_by_2(list):
+    product = 1.0
+    for item in list:
+        product *= 2
+    return product
```

Comment:
Did you want to really return 2^# of elements in the list?

Reply:
Fixed function